Commands/Commands/utils.js

```diff
@@ -79,7 +78,7 @@ commands.stats = {
           inline: true
         }
       )
-      msg.channel.sendMessage('', false, {
+      msg.channel.openDM('', false, {

```

Comment:
Wow, fail on my part, completely forgot about it. (I also wasn't expecting a review so suddenly)